### ISSUE TITLE
Error determining whether a variable is optional

### ISSUE THREAD

**Issue by achan** [krzysztofzablocki/Sourcery#99]
Not sure if I'm doing this right... but if I try to create a template using `isOptional` the sourcery script crashes:

```
2017-01-04 01:21:24.154 Sourcery[48549:19632156] *** Terminating app due to uncaught exception 'NSUnknownKeyException', reason: '[<Sourcery.Variable 0x7ff5cf9340a0> valueForUndefinedKey:]: this class is not key value coding-compliant for the key isOptional.'
```

...

```
Generating code.../Users/achan/repos/Noosh/Pods/Sourcery/bin/sourcery: line 6: 48549 Abort trap: 6           "${parent_path}"/Sourcery.app/Contents/MacOS/Sourcery "$@"
Command /bin/sh failed with exit code 134
```

I'm using Cocoapods to install Sourcery (0.5.0) with a template like this:

```stencil
{% for type in types.implementing.MyProtocol %}
  {% for variable in type.variables %}
    {{ variable.isOptional }}
  {% endfor %}
{% endfor %}
```

If I replace `{{ variable.isOptional }}` with `{{ variable.isStatic }}`, the script passes and I get a file with a bunch of 0s.

**Comment by krzysztofzablocki** [comment]
you are doing it right, there is a bug on 0.5.0 because we used POP and that's not working with KVC, I already fixed it on master and will be available in next release, for now you can just do `variable.typeName.isOptional`

**Comment by achan** [comment]
👍 Thanks for the quick reply and great work to everyone involved. Loving the project so far!

**Comment by krzysztofzablocki** [comment]
@achan should be fixed in the newest `0.5.1` release, let me know ;)

**Comment by achan** [comment]
Confirmed 👍
